client/galaxy/scripts/mvc/collection/list-collection-creator.js

```diff
@@ -278,9 +278,9 @@ var ListCollectionCreator = Backbone.View.extend(BASE_MVC.LoggableMixin)
 
         /** describe what is wrong with a particular element if anything */
         _isElementInvalid: function(element) {
-            if (element.history_content_type !== "dataset") {
-                return _l("is not a dataset");
-            }
+            // if (element.history_content_type !== "dataset") {
+            //     return _l("is not a dataset");
+            // }
```

Comment:
Ha! That was simple it seems - very nice!

Reply:
I improved that so it respects data coming from datalibrary